### ISSUE TITLE
[BUG] - Hydraulic pressure value should be rounded off 

### ISSUE THREAD

**Issue by fugotaro** [invernyx/md11-bugs#1014]
### Describe the bug

Hydraulic pressure value should be rounded off 

### Expected behaviour

Last digit should always show zero to my knowledge 

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

No

### Agreement

- [X] Yes

**Comment by GenericNerd** [comment]
I do not believe this to be the case, as we found that pilots reported that the value naturally fluctuates. Can you provide your evidence for this, along with a screenshot of what you are seeing?

**Comment by fugotaro** [comment]
![Schermopname (453)](https://github.com/invernyx/md11-bugs/assets/150957609/d0f1e412-10a9-4bb3-8569-7d91b4aaac12)

What you're saying is correct but by rounded off I mean the value being 24, 9 and 2963 for instance but that should display 20, 10 and 2960

The version before this one (or two now) displayed it correctly 

![image](https://github.com/invernyx/md11-bugs/assets/150957609/0b5d91c0-78d3-4d8b-8bb5-2c097884e171)

**Comment by joon44** [comment]
> ![Schermopname (453)](https://private-user-images.githubusercontent.com/150957609/320953964-d0f1e412-10a9-4bb3-8569-7d91b4aaac12.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2OTI4MDIsIm5iZiI6MTcxMjY5MjUwMiwicGF0aCI6Ii8xNTA5NTc2MDkvMzIwOTUzOTY0LWQwZjFlNDEyLTEwYTktNGJiMy04NTY5LTdkOTFiNGFhYWMxMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOVQxOTU1MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MzkyODBhYzU0YjBlZWUwOWU3Njc2ZWVkYWQzOWIxOGIwYjhmNjM4MzM5YmExNTllYmI5YjU3YjM2NGRjYjAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.aSHJGrlIBAJQzDZ1q9RNg1-1KxZ_ujQY_RRkdfQ_-aA)
> 
> What you're saying is correct but by rounded off I mean the value being 24, 9 and 2963 for instance but that should display 20, 10 and 2960
> 
> The version before this one (or two now) displayed it correctly
> 
> ![Schermopname (353)](https://private-user-images.githubusercontent.com/150957609/320955927-3a0876e8-4b84-461c-8a31-a335820f97c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2OTI4MDIsIm5iZiI6MTcxMjY5MjUwMiwicGF0aCI6Ii8xNTA5NTc2MDkvMzIwOTU1OTI3LTNhMDg3NmU4LTRiODQtNDYxYy04YTMxLWEzMzU4MjBmOTdjOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOVQxOTU1MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOTFiMTQ0ODY3NmEwNDZhNWE3MzI0ZDA4ZjZiYWFiNWUxNDcyODc4M2Q0NGFhYmZhYmI5ZGRkMWVjNGNhNWI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yLnn--e8MaMxPo5HhX-l8NYKgn3WGtiiEZPINtLZXGQ)

Also the current yellow box is a bit too big vertically and should be lower so that the digits vertically fill the box fully
